File: Triangle shrinking downwards/Triangle_shrinking_downwards.cpp
class Solution {
  public:
    string triDownwards(string s) {
        // in this problem we have to replace character with dot(.)
        int n = s.size();
        string ans = "";
        for(int i =0; i<n; i++) {
            // add dot(.) first;
            for(int j =0; j<i; j++) {
                ans += ".";
            }
            // add remain character
            for(int j =i; j<n; j++) {
                ans += s[j];
            }
        }
        return ans;
    }
    
};

File: Triangle shrinking downwards/Triangle_shrinking_downwards.py
class Solution:
    def triDownwards(self, s):
        # in this problem we have to replace character with dot(.)
        n = len(s)
        ans = "";
        for i in range(0,n):
            # add dot(.) first;
            for j in range(0,i):
                ans += ".";
            # add remain character
            for j in range(i,n):
                ans += s[j];
        return ans;
